.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
 
       - name: Install python dependencies
         run: |
-          pip install -U pip wheel setuptools
+          pip install -U "pip==20.2.4" wheel setuptools
```

Comment:
Bind pip to v20.2.4 as it was causing dependency issues

Reply:
This change is in conflict with https://github.com/openwisp/openwisp-radius/pull/245, I'll try to remove it shortly.